modules/client-configuration/pages/clients-rocky.adoc

```diff
@@ -97,7 +95,7 @@ This is due to the way that {rocky} manages their repositories.
 {rocky} removes older version of packages when a new version is released, while {productname} keeps all of them, regardless of age.
 ====
 
-If you are using modular channels, you must enable the Python 3.6 module stream on the client.
+If you are using modular channels with {rocky} 8, you must enable the Python 3.6 module stream on the client.
```

Comment:
Why is only Rocky 8 specifically mentioned here? How or are things different for Rocky 9?

Reply:
iirc, this has something to do with the Salt bundle. But I admit, I'm not 100% sure...

Reply:
I don't think this is needed. There's no such module on Rocky 9. @vzhestkov can confirm, just in case.

Reply:
I don't know the reason of this note. Salt bundle will not use any of Python 3.6 modules from the system. Maybe ask Can, if I'm not wrong he was developing modular support.

Reply:
somewhere else Can said, from devel POV both version are handled the same.  Thus I assume that we can drop the statement about Python 3.6 here completely.  If nobody vetoes, I'll remove it soon.